python-ufonormalizer/PKGBUILD

```diff
@@ -4,37 +4,38 @@
 pkgname=python-ufonormalizer
 _pyname=${pkgname#python-}
 pkgver=0.6.1
-pkgrel=3
-pkgdesc='A tool that will normalize XML and other data inside of a UFO'
+pkgrel=6
+pkgdesc="A tool that will normalize XML and other data inside of a UFO"
 arch=(any)
-url="https://github.com/unified-font-object/ufoNormalizer"
-license=(BSD)
-depends=(python)
-makedepends=(python-{build,installer,wheel}
-             python-setuptools-scm)
+url="https://github.com/unified-font-object/$_pyname"
+license=('BSD-3-Clause')
+depends=(python
+         python-setuptools-scm)
+makedepends=(python-{build,installer,wheel})
+checkdepends=(python-pytest)
 _archive="$_pyname-$pkgver"
-source=("https://files.pythonhosted.org/packages/source/${_pyname::1}/$_pyname/$_archive.zip")
+source=("$_archive.zip::https://files.pythonhosted.org/packages/source/u/$_pyname/$_archive.zip")
```

Comment:
Again, it is not necessary to rename the output file whet it already has the right name.